ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -39,6 +39,7 @@ component:
   dimmer: "A device which can reduce lighting output of a lamp (thus 'dimming' the light)."
   dishwasher: "Device that washes dishes."
   dryer: "A component used for drying clothes."
+  elevator: "Vertical transport system."
```

Comment:
"Machine to vertically transport people or items between floors in the building."